### DISCUSSION TITLE
Poll: Icon size on chips

### DISCUSSION THREAD

**Discussion by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#1049] [Polls]
## I'm looking only for the icon size on the chips

I got a lot of feedback about icon size on the chips, so I'd like to run a poll.

| Option 1 | Option 2 |
| :--: | :--: |
| Font 8 (as it is today) | Font 9 (increased icon size) |
| ![EU font 8](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/dfb79856-8456-443c-b4de-a955be8e4561) | ![EU font 9](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/268ec945-94dc-4f17-be94-8abd691ef2ed)  |
| ![US font 8](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/a9eb3578-901a-444f-9d52-3909f2aa4f34) | ![US font 9](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/e46bac5e-8a84-4cfe-a01d-159042828350) |

**Comment by lordhelmet77** [comment]
I'd rather more space for Entity 3 in the upper right, for something like the status of my washing machine.

**Comment by ra-kal** [comment]
Bigger please! The icons are almost indistinguishable from a half meter away. I would also like the wifi icon to be on the chips row.

**Reply by edwardtfn** [reply]
> I would also like the wifi icon to be on the chips row.

That row is quite full, specially in the US model... 😞

**Reply by edwardtfn** [reply]
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/83e64dfa-b2cf-4186-af2a-6f89c96e9044)

😉

**Reply by edwardtfn** [reply]
It is available on `dev` for testing... 😉
